website/source/guides/operations/index.html.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: "guides"
+page_title: "Vault Architecture - Guides"
+sidebar_current: "guides-operations"
+description: |-
+  Vault architecture guide covers Vault infrastructure discussions including
+  installation.   
+---
+
+# Vault Architecture
```

Comment:
Should this be "Architecture" or "Operations"? It looks like Operations is used everywhere else

Reply:
yep, my bad.  This should've been "Operations".